editoast/src/views/v2/timetable/stdcm.rs

```diff
@@ -83,10 +83,9 @@ pub struct STDCMRequestPayload {
     comfort: Comfort,
     /// By how long we can shift the departure time in milliseconds
     /// Deprecated, first step data should be used instead
-    #[serde(default = "default_maximum_departure_delay")]
-    #[schema(default = default_maximum_departure_delay)]
-    maximum_departure_delay: u64,
+    maximum_departure_delay: Option<u64>,
     /// Specifies how long the total run time can be in milliseconds
+    /// Deprecated, first step data should be used instead
```

Comment:
Unless I'm wrong, it's not really deprecated but reserved to developers for debugging situations (ie. `request_user.role == Role::Dev` when we'll have role checking 😁).

Reply:
This should actually be removed at the same time as `start_time` and `maximum_departure_delay` parameters, following the removal of the v1 debugging interface and the update of the v2 interface to allow a debugging mode with more inputs/outputs. Hence the `Deprecated`. I'd rather keep it likeso if it's all the same with you :)

Reply:
Fine by me, resolving :)